core/src/main/java/io/confluent/connect/storage/StorageSinkConnectorConfig.java

```diff
@@ -214,4 +253,12 @@ public Object get(String key) {
   public StorageSinkConnectorConfig(ConfigDef configDef, Map<String, String> props) {
     super(configDef, props);
   }
+
+  public AvroDataConfig avroDataConfig() {
+    Map<String, Object> props = new HashMap<>();
+    props.put(SCHEMA_CACHE_SIZE_CONFIG, get(SCHEMA_CACHE_SIZE_CONFIG));
```

Comment:
Actually, `SCHEMA_CACHE_SIZE_CONFIG` is a different key than `AvroDataConfig`'s `SCHEMAS_CACHE_SIZE_CONFIG`, so this doesn't create `AvroDataConfig` correctly. I've reported on #64, and provided a fix on #65.